src/index.html

```diff
@@ -7,7 +7,47 @@
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body >
+    <main class="card" data-qa="card">
+
+      <img
+      src="../src/images/imac.jpeg"
+      alt="main picture"
+      class="card__image">
+
+
+      <section class="card__discription">
+        APPLE A1419 iMac 27" Retina<br>5K Monoblock (MNED2UA/A)
+      </section>
+
+      <section class="card__prodact">
+        Product code: 195434
+      </section>
+
+      <section class="card__review">
+        <div class="card__review-stars">
+          <div class="card__review-stars-star"></div>
```

Comment:
Fix it everywhere
```suggestion
          <div class="stars__star"></div>
```

Reply:
<img width="651" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/ed6d4b67-e36c-4b0f-9e08-8f968e597079">
should be the separate BEM block

Reply:
Still not fixed